fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/IndexOptions.java

```diff
@@ -110,7 +110,11 @@
     /**
      * The name of the Lucene analyzer to use with a {@link IndexTypes#TEXT} index.
      */
-    public static final String TEXT_ANALYZER_NAME_OPTION = "testAnalyzerName";
+    public static final String TEXT_ANALYZER_NAME_OPTION = "textAnalyzerName";
```

Comment:
In theory, changing this value could break a meta-data that has already been created and that used the name with a typo. Given that this is only being used on an `EXPERIMENTAL` index type, we could fix this typo and move on, but we could theoretically add a new (deprecated) index option with the old name and check both in the maintainer if we really wanted to. I'm not sure it's worth it in this case.

Reply:
Agreed, as this is experimental, I think we can simply fix the typo.

Reply:
@arnaud-lacurie thank you for catching this typo!